package.json

```diff
@@ -33,21 +33,31 @@
   "homepage": "https://github.com/borisyankov/react-sparklines#readme",
   "devDependencies": {
     "babel-core": "^5.6.15",
+    "babel-jest": "*",
     "babel-loader": "^5.2.2",
     "jest-cli": "^0.4.15",
     "jsx-loader": "^0.13.2",
-    "babel-jest": "*",
     "node-libs-browser": "^0.5.2",
     "react-tools": "*",
     "webpack": "^1.10.0",
     "webpack-dev-server": "^1.10.1"
   },
-  "dependencies": {
+  "peerDependencies": {
+    "react": "^0.13.3"
```

Comment:
This was causing issues with duplicate Reacts.
I had it before, and removed it intentionally.
If it isn't messing up the new build thing, better remove it?
Or any other ideas? (we at least need a or 0.14-rc thing)